docs/user/resources/01-telemetry.md

```diff
@@ -66,6 +66,28 @@ For further examples, see the [samples](https://github.com/kyma-project/telemetr
 
 For details, see the [Telemetry specification file](https://github.com/kyma-project/telemetry-manager/blob/main/apis/operator/v1alpha1/telemetry_types.go).
 
+### Annotations
+
+Backward compatibility for internal metrics in OpenTelemetry:
+The recent version of OpenTelemetry Collector 0.119.0 introduced breaking changes for internal metrics exposed through the Prometheus endpoint and breaks the stability of internal metrics.
```

Comment:
We won't remember to remove the "recent" bit here even when it's not recent anymore. We can simply drop it now.

```suggestion
OpenTelemetry Collector 0.119.0 introduces breaking changes for internal metrics exposed through the Prometheus endpoint and breaks the stability of internal metrics. This affects Kyma Telemetry in the following ways:
```